sdks/java/io/iceberg/src/main/java/org/apache/beam/sdk/io/iceberg/IcebergWriteSchemaTransformProvider.java

```diff
@@ -160,7 +176,14 @@ public PCollectionRowTuple expand(PCollectionRowTuple input) {
 
       IcebergIO.WriteRows writeTransform =
           IcebergIO.writeRows(configuration.getIcebergCatalog())
-              .to(TableIdentifier.parse(configuration.getTable()));
+              .to(
+                  new PortableIcebergDestinations(
```

Comment:
Do we always expect the schema-transform to use "PortableIcebergDestinations" or will there be cases where we have to use the pass through (SingleTableDynamicDestinations) ?

Reply:
`PortableIcebergDestinations` also supports single table destination. Essentially if there are no curly braces in the destination string, it treats it like a single table.